sim/build.sbt

```diff
@@ -41,47 +42,35 @@ lazy val chipyardDir = if(firesimAsLibrary) {
 }
 
 lazy val chipyard      = ProjectRef(chipyardDir, "chipyard")
-lazy val chisel        = ProjectRef(workspaceDirectory / "chisel3", "chisel")
-lazy val firrtl        = ProjectRef(workspaceDirectory / "firrtl", "firrtl")
```

Comment:
Just to double-check... FIRRTL was only needed for some test sources, which are now copied into FireSim?

Reply:
The dependencies on classes in `firrtl % main` is brought it transitively through chisel. Classes in `test` are not published regardless, hence the copied sources.